boot/freeldr/freeldr/uefi.cmake

```diff
@@ -28,6 +28,9 @@ if(ARCH STREQUAL "i386")
 elseif(ARCH STREQUAL "amd64")
     #TBD
 elseif(ARCH STREQUAL "arm")
+    list(APPEND UEFILDR_ARC_SOURCE
+        arch/arm/macharm.c
+        arch/arm/debug.c)
```

Comment:
Are those files used only in *UEFI* builds of ARM32 (and not in non-UEFI builds)?

Reply:
We talked about it else where putting it here for record;
I put these together specifically for UEFI/QEMU, we can adapt them to normal freeldr if we ever fix it one day for arm.